docs/sources/tempo/api_docs/pushing-spans-with-http.md

```diff
@@ -13,14 +13,13 @@ pushing spans with HTTP/JSON from a Bash script using the [OpenTelemetry](https:
 
 ## Before you begin
 
-This procedure uses an example Docker Compose setup to run Tempo, so you do not need an existing installation. The Docker image also includes a Grafana container which will allow us to visualize traces.
+This procedure uses an example Docker Compose setup to run Tempo, so you don't need an existing installation. The Docker image also includes a Grafana container which lets you visualize traces.
 
 To use this procedure, you need to have Docker and `docker compose` installed.
 
-
 ## Start Tempo using the quick start
 
-Use the instructions in the [Quick start for Tempo documentation]({{< relref "../getting-started/docker-example" >}}) to start a local instance of Tempo and Grafana.
+Use the instructions in the [Quick start for Tempo documentation](https://grafana.com/docs/tempo/<TEMPO_SERVER>/getting-started/docker-example/) to start a local instance of Tempo and Grafana.
```

Comment:
```suggestion
Use the instructions in the [Quick start for Tempo documentation](https://grafana.com/docs/tempo/<TEMPO_VERSION>/getting-started/docker-example/) to start a local instance of Tempo and Grafana.
```

Reply:
If we're using the Magic Linking stuff, does that mean this link will be resolved to enterprise-traces instead of Tempo? This is a case where we should link to the Tempo docs only and not to GET. 

Actually, we probably shouldn't mount this doc. I'll add it to the exclude file.